solidjs-tailwind/src/components/RepoHeading/RepoHeading.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import RepoHeading from './RepoHeading';
+
+export default {
+  component: RepoHeading,
+  title: 'RepoPage/RepoHeading',
+};
+
+const Template = (args) => <RepoHeading {...args} />;
+
+export const Public = Template.bind({});
+Public.args = {
```

Comment:
It would be nice if we could find a way to make this object re-useable as just one value differs from the other